### PR TITLE
add pandanet

### PR DESCRIPTION
**Pull request by Jennal**
pandanet is a global go game server, real slow in China

### CODE REVIEW

File: data/pandanet [hunk 1]
```diff
@@ -0,0 +1,3 @@
+joyjoy.net
+igs.joyjoy.net
```
[the remark this answers is not in the document]
Comment (Jennal) [reply data/pandanet:2]: igs.joyjoy.net is the exact domain which has been used. then I would remove joyjoy.net

[the remark this answers is not in the document]
Comment (IceCodeNew) [reply data/pandanet:2]: what's the point of marking as resolved if you just click that button while you haven't resolved the problem at all?

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

**Comment by Jennal** [comment]
sorry..I don't know how to change the content of pull request. Click resolved is a mistake, I am so sorry for that.

**Review by IceCodeNew — APPROVED** [review]

**Comment by IceCodeNew** [comment]
> sorry..I don't know how to change the content of pull request. Click resolved is a mistake, I am so sorry for that.

Never mind. Thanks for your contribution.

**Pull request merged**